test/annexB/built-ins/escape/argument_types.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2020 Qu Xing. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-escape-string
+description: Input is a null, undefined or boolean
+info: |
+    B.2.1.1 escape ( string )
+
+    1. Let string be ? ToString(string).
+    ...
+---*/
+
+assert.sameValue(escape(null), 'null');
+
+assert.sameValue(escape(undefined), 'undefined');
+
+assert.sameValue(escape(true), 'true');
+
+assert.sameValue(escape(false), 'false');
```

Comment:
This could still have extended coverage here for other types.

In the same file:

- undefined (implicit): escape() === 'undefined'
- Number: escape(-0), escape(0), escape(1), escape(NaN), escape(Infinity)

In a different file:

- BigInt: escape(1n)

We have tests for objects using toString and valueOf. We can add a test for using @@toPrimitive as well, i.e.:

```js
/*---
...
features: [Symbol.toPrimitive]
---*/

var obj = {
  toString() { throw new Test262Error('this should be unreachable'); },
  valueOf() { throw new Test262Error('this should be unreachable'); },
  [Symbol.toPrimitive]() { return 'success'; }
};

assert.sameValue(escape(obj), 'success');
```

The tests could be reused for unescape.

---

This can be done in a follow up PR

Reply:
I will extend these coverages in a new PR.

Reply:
Thanks! I created #2694 for this part.